Persistence/Entities/Shipment.cs

```diff
@@ -13,6 +13,10 @@ public class Shipment : IBaseEntity
         public PostMachine DestinationMachine { get; set; }
         public long DestinationMachineId { get; set; }
         public ShipmentStatus Status { get; set; }
+        public int? SrcPmSenderUnlockCode { get; set; }
+        public int? SrcPmCourierUnlockCode { get; set; }
+        public int? DestPmSenderUnlockCode { get; set; }
+        public int? DestPmCourierUnlockCode { get; set; }
```

Comment:
Shouldn't DestPmSenderUnlockCode be named DestPmReceiverUnlockCode?

Reply:
Also, why are we using int instead of string?

Reply:
I think we also need a field to hold the shipment tracking link, since we will have to display it to the user (as mentioned in the user story).

Reply:
Yeah, it should be receiver unlock code. As I wrote in meeting summary, we decided to make the link with ID + ID hash so saving it is not necessary we can just quickly calculate it.
Why string? Won't a code be a PIN?

Reply:
Also, could you add hash checking and creation methods in the Shipment entity? Maybe it could be elsewhere but I think it has the point being placed here since it could be a db field, just we chose not to store it.

Reply:
Since a code is 4, 6 or whatever digits, int should be fine, although string is definitely an option, since the codes are not meant to be manipulated and will all be the same length. As for hash checking, I think it should reside in "usecases" (wherever that is), but I can add it here for the time being.